fe/fe-core/src/main/java/org/apache/doris/tablefunction/S3TableValuedFunction.java

```diff
@@ -68,12 +66,17 @@ public S3TableValuedFunction(Map<String, String> properties) throws AnalysisExce
         forwardCompatibleDeprecatedKeys(otherProps);
 
         String usePathStyle = getOrDefaultAndRemove(otherProps, PropertyConverter.USE_PATH_STYLE, "false");
-        boolean forceVirtualHosted = isVirtualHosted(uriStr, Boolean.parseBoolean(usePathStyle));
-        S3URI s3uri = getS3Uri(uriStr, forceVirtualHosted);
-        String endpoint = forceVirtualHosted
-                ? getEndpointAndSetVirtualBucket(s3uri, otherProps) : s3uri.getBucketScheme();
+        String forceParsingByStandardUri = getOrDefaultAndRemove(otherProps,
```

Comment:
Define the default value of `forceParsingByStandardUri` and `usePathStyle` somewhere.

Reply:
OK.